### PR TITLE
Add --check-format flag and improve help texts

### LINKED ISSUE #40: Let `-c` or some new flag take formatting into consideration
We use `cargo-sort` in CI to check if entries are sorted but it would also be super helpful to validate if it has been formatted correctly. I suppose making `-c` return a non-zero exit code would be a breaking change, so perhaps some other option would be necessary.

### PR DESCRIPTION
**Pull request by matze**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -120,35 +133,39 @@ fn _main() -> IoResult<()> {
...
+                "Returns non-zero exit code if Cargo.toml is unsorted, overrides default behavior",
             ))
             .arg(
                 Arg::with_name("print")
                     .short("p")
                     .long("print")
                     // No printing if we are running a --check
                     .conflicts_with("check")
-                    .help("prints Cargo.toml, lexically sorted, to stdout"),
+                    .help("Prints Cargo.toml, lexically sorted, to stdout"),
```
Comment (DevinR528) [inline src/main.rs:144]: I think conventionally help text is all lowercase unless there are multiple sentences. Do you mind changing back?

File: src/main.rs [hunk 2]
```diff
@@ -120,35 +133,39 @@ fn _main() -> IoResult<()> {
...
                     .long("workspace")
-                    .help("checks every crate in a workspace"),
+                    .help("Checks every crate in a workspace"),
             )
             .arg(
                 Arg::with_name("grouped")
                     .short("g")
                     .long("grouped")
-                    .help("when sorting groups of key value pairs blank lines are kept"),
+                    .help("Keep blank lines when sorting groups of key value pairs"),
```
Comment (DevinR528) [inline src/main.rs:168]: I like your wording better! Leave what you have just lowercase "Keep".

**Review by DevinR528 — CHANGES_REQUESTED** [review]
The code changes look great! Just lowercase, and I can merge. Sorry, this has taken me so long.

File: src/main.rs (hunk 2 above)
Comment (DevinR528) [reply src/main.rs:168]: (for this line and the one on 177)

File: src/main.rs (hunk 1 above)
Comment (matze) [reply src/main.rs:144]: I tried to be in line with the default texts for help and version which are
```
    -h, --help            Prints help information
    [...]
    -V, --version         Prints version information
```
but if you insist I can change back of course.

Comment (DevinR528) [reply src/main.rs:144]: Yeah, I guess stick with the capitalization, it seems like that is becoming more common.

**Comment by DevinR528** [comment]
As long as you are ready to merge I'm happy with this.

Resolves #40

**Comment by matze** [comment]
> As long as you are ready to merge I'm happy with this.

Yes from my side it's ready. And thanks for the taking the time!

**Comment by DevinR528** [comment]
Again sorry this took so long @matze

**Pull request merged**